src/MyHealth/Today.spec.tsx

```diff
@@ -55,6 +57,34 @@ describe("Today", () => {
   })
 
   describe("when the user has not checked in today", () => {
+    it("shows an error if checking in fails", async () => {
+      const showMessageSpy = showMessage as jest.Mock
+      const errorMessage = "errorMessage"
+      const addTodaysCheckInSpy = jest
+        .fn()
+        .mockResolvedValueOnce({ kind: "failure", errorMessage })
+
+      const { getByLabelText } = render(
+        <SymptomLogContext.Provider
+          value={factories.symptomLogContext.build({
+            addTodaysCheckIn: addTodaysCheckInSpy,
+          })}
+        >
+          <Today />
+        </SymptomLogContext.Provider>,
+      )
+
+      fireEvent.press(getByLabelText("Good"))
+
+      await waitFor(() => {
+        expect(showMessageSpy).toHaveBeenCalledWith(
+          expect.objectContaining({
+            message: "Sorry, we could not add todays check in",
```

Comment:
@aledustet Does this need to be translated? Also I think it wants to be `add today's check in` with an apostrophe in `today`.